wire-ios/Wire-iOS Tests/CallControllerTests.swift

```diff
@@ -156,7 +156,7 @@ final class CallControllerTests: XCTestCase, CoreDataFixtureTestHelper {
     func testThatVersionDegradationAlertIsNotPresented_WhenVoiceChannelHasNotDegradationState() {
         // GIVEN
         let callState: CallState = .established
-        sut.callConversationProvider?.priorityCallConversation = conversation
+        callConversationProvider?.priorityCallConversation = conversation
```

Comment:
I think I have fixed this recently, what's going on here? 🤔

Reply:
Maybe only on develop?

Reply:
@markusfassbender you targeted develop

Reply:
Yes I think @netbe told me to do so. So shouldn't these changes also only apply for develop?

Reply:
Since we're actively working on this branch for the Zulu-Hotfix I suggest to keep `develop` and `release/cycle-3.112` as close as possible.

Reply:
Okay, from my side it makes sense. maybe there will be a few merge conflicts with `develop`.